Share Extension/ShareViewController.swift

```diff
@@ -22,26 +21,29 @@ class ShareViewController: UIViewController {
             // Request the OPML file URL
             attachment.loadItem(forTypeIdentifier: UTType.data.identifier, options: nil) { [weak self] data, error in
                 guard let url = data as? URL else {
-                    self?.close()
                     return
                 }
 
-                // Convert the file to Data
-                guard let opmlData = try? Data(contentsOf: url) else {
-                    self?.close()
+                // Save the file to the shared group directory
+                let fileManager = FileManager.default
+                guard let container = fileManager.containerURL(forSecurityApplicationGroupIdentifier: "group.au.com.shiftyjelly.pocketcasts") else {
```

Comment:
"group.au.com.shiftyjelly.pocketcasts" can be  `SharedConstants.GroupUserDefaults.groupContainerId`

Reply:
Done in 65ca83342d2a5cc54485016399e75af175a0dc03